### PR TITLE
Event pictures

### PR DESCRIPTION
**Pull request by keviiinmart**
Debugging changes were added to this pull request. I am not sure if they should be here.

otherwise picture form has been added to the events

### CODE REVIEW

File: thetatauCMT/events/models.py [hunk 1]
```diff
@@ -11,6 +13,12 @@
 from chapters.models import Chapter
 
 
+def get_event_upload_event(instance, filename):
```
Comment (VenturaFranklin) [inline thetatauCMT/events/models.py:16]: This function doesn't seem to be used and is unnecessary.
Along with the "from django.conf import settings" I think is also unnecessary at the top of the file, please remove them.

File: thetatauCMT/templates/events/event_form.html [hunk 2]
```diff
@@ -8,10 +11,28 @@ <h1>Update Event: {{ event.name }}</h1>
   <form class="form-horizontal" method="post" action="{% url 'events:update' event.id %}">
     {% csrf_token %}
     {{ form|crispy }}
+    <h2>Pictures</h2>
```
Comment (VenturaFranklin) [inline thetatauCMT/templates/events/event_form.html:14]: I thought we were not going to put pictures in the update form. Just in the main create form? Does this even work? I thought it didn't. If not, you should remove it.

**Review by VenturaFranklin — CHANGES_REQUESTED** [review]
Please remove the unused parts of the code. See two comments.

File: thetatauCMT/templates/events/event_form.html (hunk 2 above)
Comment (keviiinmart) [reply thetatauCMT/templates/events/event_form.html:14]: Viewing the pictures works but adding and removing pictures does not.

**Pull request merged**